src/index.html

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
+Hello, world!
```

Comment:
Add full HTML template:
Usually it is done by typing `!` symbol in an empty HTML document and pressing `Tab`.
Then you can insert your content inside `<body>` tag

![image](https://user-images.githubusercontent.com/60885495/111953029-b648b580-8aee-11eb-8d6e-a90a077a6b2c.png)

Reply:
I did it